src/main/java/nextstep/subway/fare/domain/DistanceFarePolicy.java

```diff
@@ -0,0 +1,46 @@
+package nextstep.subway.fare.domain;
+
+import nextstep.subway.line.domain.Distance;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DistanceFarePolicy {
+    private static final Fare BASIC_FARE = Fare.from(1250);
+    private static final Distance BASIC_DISTANCE = Distance.from(10);
+    private static final Distance PREMIUM_DISTANCE = Distance.from(50);
+
+
+    public Fare calculate(Distance distance) {
+        if (distance.compareTo(BASIC_DISTANCE) != 1) {
```

Comment:
```suggestion
        if (distance.isLessThan(BASIC_DISTANCE)) {
```

위와 같이 조금 더의미전달을 강화해보는것은 어떤가요?

Reply:
의미가 더 잘 드러나도록 변경했습니다.